src/cargo/core/resolver/encode.rs

```diff
@@ -281,14 +281,15 @@ impl<'a, 'cfg> Encodable for WorkspaceResolve<'a, 'cfg> {
 
         let root = self.ws.members().max_by_key(|member| {
             member.name()
-        }).unwrap().package_id();
+        }).map(Package::package_id);
```

Comment:
Hm were the changes in this file required to get tests to pass? Or just cleaning things up?